sites/public/src/pages/listing/ext/[id].tsx

```diff
@@ -104,5 +109,6 @@ export async function getServerSideProps(context: {
       googleMapsApiKey: runtimeConfig.getGoogleMapsApiKey() || null,
       googleMapsMapId: runtimeConfig.getGoogleMapsMapId() || null,
     },
+    revalidate: Number(process.env.cacheRevalidate),
```

Comment:
While testing this locally I notice different headers than what I get for core and Netlify deploys. Everything is working correctly but the cache control uses the `s-maxage` (45 is what I set as an environment variable). And the `x-nextjs-cache` value is always correct

```
HTTP/1.1 200 OK
x-nextjs-cache: HIT
X-Powered-By: Next.js
ETag: "2jxyuh7mxz13h9"
Content-Type: text/html; charset=utf-8
Cache-Control: s-maxage=45, stale-while-revalidate
Vary: Accept-Encoding
Content-Encoding: gzip
Date: Mon, 09 Jun 2025 19:32:03 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Transfer-Encoding: chunked
```